app/uk/gov/hmrc/gform/sharedmodel/formtemplate/ComponentType.scala

```diff
@@ -485,11 +485,26 @@ object MiniSummaryRow {
   implicit val format: Format[MiniSummaryRow] = derived.oformat()
 }
 
-case class MiniSummaryList(rows: List[MiniSummaryRow]) extends ComponentType
+case class MiniSummaryList(rows: List[MiniSummaryRow], displayInSummary: DisplayInSummary = IsNotDisplayInSummary)
+    extends ComponentType
 object MiniSummaryList {
   implicit val format: Format[MiniSummaryList] = derived.oformat()
 }
 
+sealed trait DisplayInSummary
+
+case object IsDisplayInSummary extends DisplayInSummary
+case object IsNotDisplayInSummary extends DisplayInSummary
```

Comment:
Move this into companion object and let's called them `Yes` and `No` only
```suggestion
object DisplayInSummary {
  case object Yes extends DisplayInSummary
  case object No extends DisplayInSummary

  ...
}

```

Reply:
Updated @josef-vlach